bin/configure-fastly.js

```diff
@@ -46,18 +47,42 @@ async.auto({
         var passStatement = fastlyConfig.negateConditionStatement(notPassStatement);
         
         // For a non-pass condition, point backend at s3
-        var backendCondition = fastlyConfig.setBackend(
-            'F_s3',
-            S3_BUCKET_NAME,
-            notPassStatement
-        );
-        // For a pass condition, set forwarding headers
-        var forwardCondition = fastlyConfig.setForwardHeaders(passStatement);
```

Comment:
Since we've inlined these two functions, we should remove their definitions to avoid cruft.

Reply:
Good point. Removing those.